src/test/java/org/mockito/internal/creation/bytebuddy/InlineByteBuddyMockMakerTest.java

```diff
@@ -47,6 +45,8 @@ public void should_create_mock_from_final_class() throws Exception {
 
     @Test
     public void should_create_mock_from_final_class_in_the_JDK() throws Exception {
+        assumeTrue(ClassFileVersion.ofThisVm().isLessThan(ClassFileVersion.JAVA_V9)); // Change when ByteBuddy has ASM6
```

Comment:
This will be a conflict with current master: https://github.com/mockito/mockito/blob/master/src/test/java/org/mockito/internal/creation/bytebuddy/InlineByteBuddyMockMakerTest.java#L42 Let's choose one way or another. I am fine with either way, no specific preference.